cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc

```diff
@@ -72,10 +72,10 @@ These IP addresses will be added to the https://docs.paloaltonetworks.com/prisma
 
```

Comment:
@rcaballeromx @Pubs-MV There is no blurb for this. Maybe we should comment this out?

Reply:
are you referring to CWP-47766, as having no blurb? @divyabhushan